apps/web-giddh/src/app/subscription/subscription.component.ts

```diff
@@ -1,769 +1,372 @@
-    public subscribedCompaniesInProgress$ = this.componentStore.select(state => state.subscribedCompaniesInProgress);
-    /* This will hold list of subscriptions */
-    public subscriptions: SubscriptionsUser[] = [];
-    /* This will hold the companies to use in selected company */
-    public selectedCompany: any;
-    /** This will use for active company */
-    public activeCompany: any = {};
-    /** True if subscription will move */
-    public subscriptionMove: boolean = false;
-    /** Holds Store Buy Plan Success observable*/
-    public buyPlanSuccess$ = this.componentStore.select(state => state.buyPlanSuccess);
-    /** This will use for open window */
-    private openedWindow: Window | null = null;
-
-
-    constructor(public dialog: MatDialog,
-        private changeDetection: ChangeDetectorRef,
-        private generalService: GeneralService,
-        private componentStore: SubscriptionComponentStore,
-        private store: Store<AppState>,
-        private formBuilder: FormBuilder,
-        private readonly componentStoreBuyPlan: BuyPlanComponentStore,
-        private generalActions: GeneralActions,
+    /** This will hold toggle buttons value and size */
+    public bootstrapToggleSwitch = BootstrapToggleSwitch;
+    /* Holds Mat Table Columns*/
+    public displayedColumns: string[] = ['ipaddress', 'signindate', 'signintime', 'duration', 'agent', 'action'];
+    /** Holds Active Tab Index */
+    public activeTabIndex: number = 0;
+    /** Holds Tab Name */
+    private tabName = ['auth-key', 'mobile-number', 'session', 'subscription'];
+    /** Holds True if API calling in progress in old subscription page */
+    public isSubscriptionLoading: boolean = false;
+    /** Holds subscription id */
+    public subscriptionId: string = '';
+
+    constructor(private store: Store<AppState>,
+        private toasty: ToasterService,
+        private loginService: AuthenticationService,
+        private loginAction: LoginActions,
         private router: Router,
-        private toasterService: ToasterService
-    ) {
-        this.store.dispatch(this.generalActions.openSideMenu(true));
+        private sessionAction: SessionActions,
+        private route: ActivatedRoute,
+        private breakPointObservar: BreakpointObserver,
+        private generalActions: GeneralActions,
+        private changeDetectionRef: ChangeDetectorRef,
+        private clipboardService: ClipboardService) {
+        this.contactNo$ = this.store.pipe(select(s => {
+            if (s.session.user) {
+                return s.session.user.user.contactNo;
+            }
+        }), takeUntil(this.destroyed$));
+        this.countryCode$ = this.store.pipe(select(s => {
+            if (s.session.user) {
+                return s.session.user.countryCode;
+            }
+        }), takeUntil(this.destroyed$));
+
+        this.isAddNewMobileNoInProcess$ = this.store.pipe(select(s => s.login.isAddNewMobileNoInProcess), takeUntil(this.destroyed$));
```

Comment:
don't use single character variable

Reply:
Resolved but this is an old subscription